packages/wmr/src/plugins/wmr/client.js

```diff
@@ -31,6 +33,15 @@ function connect(needsReload) {
 
 	ws.addEventListener('message', handleMessage);
 	ws.addEventListener('error', handleError);
+	ws.addEventListener('close', reconnect);
```

Comment:
We could also listen for something like focussing the browser again

Reply:
Yes, that's a great idea. Thtat's what I'm using to run my code for waking up ios wrapper app

Reply:
since the exponential dropoff with 30s limit will not solve the case for a sleep longer than 30s, but visibilitychange certainly would.

Thanks guys!